src/api/lib/backend/api/cloud.rb

```diff
@@ -0,0 +1,28 @@
+module Backend
+  module Api
+    module Cloud
+      extend Backend::ConnectionHelper
+      # Triggers a cloud upload job
+      # @return [String]
+      def self.upload(project_name, package_name, repository_name, architecture_name, file_name, user, region_name, target_name = 'ec2')
+        data = user.ec2_configuration.attributes.except('id', 'created_at', 'updated_at').merge(region: region_name).to_json
+        params = {
+          project:    project_name,
+          package:    package_name,
+          repository: repository_name,
+          arch:       architecture_name,
+          filename:   file_name,
+          target:     target_name,
+          user:       user.login
```

Comment:
Do you know why the backend needs the user's login? I thought it had no knowledge of users? @mlschroe

Reply:
Actually the user parameter is ignored for now. I guess @mlschroe implemented it for later use (e.g. when we need to introduce encryption etc)